README.md

```diff
@@ -1,127 +1,122 @@
+Step 2: Clone this repository to your local machine
 
-If installing templates from a ZIP or from a directory and a required component to make them is not already installed, run
+Step 3:  Run the setup script
+ - **Windows:** [setup.cmd](https://github.com/dotnet/templating/blob/master/setup.cmd)
+ - **Mac/Linux**: [setup.sh](https://github.com/dotnet/templating/blob/master/setup.sh) 
 
-    dotnet new3 --install-component {the ID of the NuGet package containing the required component}
+When running the setup script, the existing built in command `dotnet new` will be preserved. A new command `dotnet new3` will be enabled which allows you to create
+files with the latest Template Engine.
 
-#Basic Commands
-##Showing help
+Step 4: That's it! Now you can run `dotnet new3`.
+
+For example, here is the result of running `dotnet new3 --help` on a Mac (_truncated to save space here_).
+
+```bash
+$ dotnet new3 --help
+Template Instantiation Commands for .NET Core CLI.
+
+Usage: dotnet new3 [arguments] [options]
+
+Arguments:
+  template  The template to instantiate.
+
+<truncated>
+```
+
+# Installing templates
+
+Templates can be installed from packages in any NuGet feed, directories on the file system or ZIP type archives (zip, nupkg, vsix, etc.)
+To install a new template use the command:
+
+    dotnet new3 -i {the path to the folder containing the templates}
+
+# Basic Commands
+## Showing help
 
     dotnet new3 --help
     dotnet new3 -h
     dotnet new3
 
-##Listing templates
+## Listing templates
 
     dotnet new3 --list
     dotnet new3 -l
     dotnet new3 mvc -l            Lists all templates containing the text "mvc"
 
-##Template parameter help
-
-    dotnet new3 MvcWebTemplate --help
-    dotnet new3 MvcWebTemplate -h
-
-##Template creation
+## Template parameter help
 
-    dotnet new3 MvcWebTemplate --name MyProject --directory --ParameterName1 Value1 --ParameterName2 Value2 ... --ParameterNameN ValueN
-    dotnet new3 MvcWebTemplate -n MyProject -d --ParameterName1 Value1 --ParameterName2 Value2 ... --ParameterNameN ValueN
+    dotnet new3 mvc --help
+    dotnet new3 mvc -h
 
-##Favoriting templates
-Creates a shortcut (t1 in the example) for a template, after creating an alias, you can use it instead of the template name
+## Template creation
 
-    dotnet new3 MvcWebTemplate --alias t1
-    dotnet new3 MvcWebTemplate -a t1
+    dotnet new3 MvcWebTemplate --name MyProject --output src --ParameterName1 Value1 --ParameterName2 Value2 ... --ParameterNameN ValueN
+    dotnet new3 MvcWebTemplate -n MyProject -o --ParameterName1 Value1 --ParameterName2 Value2 ... --ParameterNameN ValueN
 
```

Comment:
The -o flag is for the output directory, so needs to be `-o <path>` (if that's what you're trying to do here)